js/components/countDownClock.js

```diff
@@ -156,7 +156,8 @@ var CountDownClock = React.createClass({
       "clickedVideo" : this.props.upNextInfo.upNextData,
       "custom" : {"source": CONSTANTS.SCREEN.UP_NEXT_SCREEN}
     };
-    this.props.controller.sendDiscoveryClickEvent(eventData);
+    var isAutoUpNext = true;
+    this.props.controller.sendDiscoveryClickEvent(eventData, isAutoUpNext);
   },
```

Comment:
@chibzu, the problem was that postroll did not play if upNext timer was up. 
For Discover, we first see the postroll and only then the Discover screen with the timer appear, so the issue does not occur.